.env.example

```
@@ -86,6 +86,21 @@ NUXT_MCP_ADMIN_TOKEN=
 # Local dev and Vercel preview default to slack/nuxi-preview; prod uses slack/nuxi.
 SLACK_CONNECTOR=
 
+# Discord channel (eve service only) — see layers/nuxi/README.md
+# Discord application from https://discord.com/developers/applications
+# (enable the Message Content privileged intent on the Bot tab)
+DISCORD_BOT_TOKEN=
+DISCORD_PUBLIC_KEY=
+DISCORD_APPLICATION_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_APPLICATION_ID key should go before the DISCORD_BOT_TOKEN key

(UnorderedKey)